### PR TITLE
improv(Dropdown) wrap dropdown content to have stable nodes on extern…

### PR DESCRIPTION
**Pull request by sergiuelisei**
https://etvas.atlassian.net/browse/ETV-5707

### CODE REVIEW

File: src/Dropdown/Dropdown.jsx [hunk 1]
```diff
@@ -207,7 +207,8 @@ const Dropdown = ({
           error={error}
           tinted={tinted}
           onClick={toggleDropdown}>
-          {isEmpty ? placeholder : displayValue}
+          {isEmpty && placeholder && <Span>{placeholder}</Span>}
+          {displayValue && <Span>{displayValue}</Span>}
```
Comment (alsofronie) [inline src/Dropdown/Dropdown.jsx:211]: Please use `TextWrapper` or `SpanWrapper`, just `Span` looks weird to me and it's not something we are using in this codebase.

**Review by alsofronie — APPROVED** [review]
Please resolve the comments

File: src/Dropdown/Dropdown.jsx (hunk 1 above)
Comment (sergiuelisei) [reply src/Dropdown/Dropdown.jsx:211]: The condition is a little bit trickier than that according to my testing, they had to be in different nodes

I will close this PR for now, I can't reproduce the bug anymore

**Comment by sergiuelisei** [comment]
I will close this PR for now, I can't reproduce the bug anymore

**Pull request closed without merging**